data_steward/cdr_cleaner/cleaning_rules/clean_height_weight.py

```diff
@@ -16,18 +16,24 @@
     MeasurementRecordsSuppression)
 # Project imports
 from common import MEASUREMENT, JINJA_ENV
-from constants.bq_utils import WRITE_TRUNCATE, WRITE_APPEND
+from constants.bq_utils import WRITE_APPEND
 from constants.cdr_cleaner import clean_cdr as cdr_consts
+from resources import fields_for
 
 LOGGER = logging.getLogger(__name__)
 
-ISSUE_NUMBERS = ['DC-416', 'DC-701']
+ISSUE_NUMBERS = ['DC-416', 'DC-701', 'DC2456']
 
 HEIGHT_TABLE = 'height_table'
 WEIGHT_TABLE = 'weight_table'
 NEW_HEIGHT_ROWS = 'new_height_rows'
 NEW_WEIGHT_ROWS = 'new_weight_rows'
 
+WEIGHT_CONCEPT_IDS = [3025315, 3013762, 3023166]
+HEIGHT_CONCEPT_IDS = [3036277, 3023540, 3019171]
+
+MEASUREMENT_FIELDS = [field.get('name') for field in fields_for(MEASUREMENT)]
```

Comment:
Thanks for making this change, by this we don't have to worry about future schema changes.